org.adempiere.ui.zk/WEB-INF/src/org/adempiere/webui/desktop/DashboardController.java

```diff
@@ -422,7 +422,7 @@ public  boolean render(Component content, MDashboardContent dc, DashboardRunnabl
 			if (dc.isEmbedReportContent()) 
 			{
 				String processParameters = dc.getProcessParameters();
-				embedReport(content, AD_Process_ID, processParameters);
+				embedReport(content, AD_Process_ID, processParameters, false);
```

Comment:
one way to reduce the changes further is to remove the isrefresh parameter from embedreport(..) method and grab the iframe reference here. after the embedreport() call, iframe will be the last children of "content". With the iframe local variable added here, the btn event listener below can be simplify further to btn.addEventListener(Events.ON_Click, e -> iframe.setContent(generateReport(AD_Process_ID, parameters)))